sdk/communication/Azure.Communication.MediaComposition/tests/samples/Sample1_MediaCompositionClient.cs

```diff
@@ -222,33 +221,33 @@ private async Task<Response<MediaComposition>> CreateMediaCompositionHelper(Medi
                 ["jill"] = new ParticipantInput
                 (
                     id: new MicrosoftTeamsUserIdentifier("f3ba9014-6dca-4456-8ec0-fa03cfa2b7b7"),
```

Comment:
Same with all these instances of the user if we are switching the group call, let's keep it consistent

Reply:
I searched and switched all to acsGroupCall. Pushed new changes.